ruby/example_code/lambda/README.md

```diff
@@ -1,3 +1,92 @@
+# Lambda code examples for the AWS SDK for Ruby
+
+## Overview
+
+These code examples demonstrate how to use the AWS SDK for Ruby (v3) to manage and invoke
+AWS Lambda functions.
+
```

Comment:
New paragraph (service blurb from [Lambda landing page](https://docs.aws.amazon.com/lambda/)): 

With AWS Lambda, you can run code without provisioning or managing servers. You pay only for the compute time that you consume—there's no charge when your code isn't running.